resources/knative-eventing/charts/knative-eventing/templates/pre-upgrade-job.yaml

```diff
@@ -57,6 +56,7 @@ data:
       subscriptions.eventing.knative.dev \
       clusterchannelprovisioners.eventing.knative.dev \
       channels.eventing.knative.dev --ignore-not-found
+    kubectl scale -n kyma-system deploy/event-bus-subscription-controller --replicas=0 || true
```

Comment:
I still believe that thing is shaky. Please use `delete` instead of `scale` we use it in all other places.
See #5886

Reply:
True that. I changed it.